src/routes/(dashboard)/PublisherDashboard.svelte

```diff
@@ -27,23 +28,28 @@
     import { filterBoolean } from '$lib/utils/array';
     import CenteredSpinnerFullScreen from '$lib/components/CenteredSpinnerFullScreen.svelte';
     import ErrorMessage from '$lib/components/ErrorMessage.svelte';
+    import { ResourceContentVersionReviewLevel } from '$lib/types/resources';
 
     enum Tab {
         myWork = 'my-work',
         reviewPending = 'review-pending',
         myProjects = 'my-projects',
+        community = 'community',
     }
 
     let assignedContents: ResourceAssignedToSelf[] = [];
     let reviewPendingContents: ResourcePendingReview[] = [];
     let assignedProjects: Project[] = [];
+    let communityPendingContents: ResourcePendingReview[] = [];
     let currentAssignedContents: ResourceAssignedToSelf[] = [];
     let currentReviewPendingContents: ResourcePendingReview[] = [];
     let currentAssignedProjects: Project[] = [];
+    let currentCommunityPendingContents: ResourcePendingReview[] = [];
 
     const sortAssignedResourceData = createPublisherDashboardMyWorkSorter();
     const sortPendingData = createPublisherDashboardReviewPendingSorter();
     const sortAssignedProjectData = createPublisherDashboardProjectsSorter();
+    // todo sort community
```

Comment:
This feels like it could be done in this ticket alongside the reset unless you have other reasons to do a follow-up?

Reply:
Yes, good catch. I'll push this shortly

Reply:
Since the sorter functionality is the same as the Review Pending tab, there was no need to write a new one for the Community Pending tab